src/Workspaces/Core/Portable/Editing/SyntaxGenerator.cs

```diff
@@ -299,6 +299,19 @@ public SyntaxNode ParameterDeclaration(IParameterSymbol symbol, SyntaxNode initi
                     setAccessorStatements);
         }
 
+        public SyntaxNode WithAccessorDeclarations(SyntaxNode declaration, params SyntaxNode[] accessorDeclarations)
+            => WithAccessorDeclarations(declaration, (IEnumerable<SyntaxNode>)accessorDeclarations);
+
+        public abstract SyntaxNode WithAccessorDeclarations(SyntaxNode declaration, IEnumerable<SyntaxNode> accessorDeclarations);
+
+        public abstract SyntaxNode GetAccessorDeclaration(
```

Comment:
Events are less necessary.  There are already ways to generate a normal field style event, as well as a property style event.  And with events you can't put custom accessibility on the accessor.